src/js/Renderer.js

```diff
@@ -457,10 +457,12 @@ define([
       var body = [];
 
       for (var i in keys) {
-        body.push(
-          '<div class="' + keyClass + 'key">' + keys[i].kbd + '</div>' +
-          '<div class="' + keyClass + 'name">' + keys[i].text + '</div>'
-          );
+        if (keys.hasOwnProperty(i)) {
```

Comment:
`for-in` loops iterate over all properties of an object--even inherited or prototypical ones.  This can cause problems.  Take this for example:

```javascript

Array.prototype.foo = 'bar';

var arr = [1, 2, 3];

for (var i in arr) {
  console.log(arr[i]);
}
//=> 1
//=> 2
//=> 3
//=> bar

for (var i in arr) {
  if (arr.hasOwnProperty(i)) {
    console.log(arr[i]);
  }
}
//=> 1
//=> 2
//=> 3
```

The same goes for objects:

```javascript
Object.prototype.foo = 'bar';

var obj = {a: 1, b: 2, c: 3}

for (var i in obj) { console.log(obj[i]); }
//=> 1
//=> 2
//=> 3
//=> bar

for (var i in obj) {
  if (obj.hasOwnProperty(i)) {
    console.log(obj[i]);
  }
}
//=> 1
//=> 2
//=> 3
```

Now consider if we're using objects with constructors:

```javascript
function Foo (a, b, c) {
  this.a = a;
  this.b = b;
  this.c = c;
}

Foo.prototype.printAll = function () {
  for (var i in this) {
    console.log(this[i]);
  }
}

var foo = new Foo(1, 2, 3);

foo.printAll();
//=> 1
//=> 2
//=> 3
//=> [Function]
//=> bar
// (guess why you shouldn't monkeypatch Object ;)
```